pkg/importer/registry-datasource.go

```diff
@@ -165,14 +166,20 @@ func getImageFileName(dir string) (string, error) {
 }
 
 func createCertificateDir(registryCertDir string) (string, error) {
-	allCerts := "/all_certs"
-	err := os.MkdirAll(allCerts, 0777)
+	allCerts := "/tmp/all_certs"
```

Comment:
just double checking that /tmp is the correct place for this.
some distros treat this differently, maybe this gets cleaned up? `/var/tmp` is another option